include/aikido/constraint.hpp

```diff
@@ -1,27 +1,63 @@
 #include "constraint/CartesianProductProjectable.hpp"
 #include "constraint/CartesianProductSampleable.hpp"
 #include "constraint/CartesianProductTestable.hpp"
-#include "constraint/CollisionFree.hpp"
 #include "constraint/CyclicSampleable.hpp"
 #include "constraint/Differentiable.hpp"
 #include "constraint/DifferentiableIntersection.hpp"
 #include "constraint/DifferentiableSubspace.hpp"
 #include "constraint/FiniteSampleable.hpp"
-#include "constraint/FrameDifferentiable.hpp"
-#include "constraint/FramePairDifferentiable.hpp"
-#include "constraint/FrameTestable.hpp"
-#include "constraint/InverseKinematicsSampleable.hpp"
-#include "constraint/JointStateSpaceHelpers.hpp"
 #include "constraint/NewtonsMethodProjectable.hpp"
 #include "constraint/Projectable.hpp"
 #include "constraint/RejectionSampleable.hpp"
 #include "constraint/Sampleable.hpp"
 #include "constraint/Satisfied.hpp"
-#include "constraint/TSR.hpp"
 #include "constraint/Testable.hpp"
 #include "constraint/TestableIntersection.hpp"
+#include "constraint/dart/CollisionFree.hpp"
+#include "constraint/dart/FrameDifferentiable.hpp"
+#include "constraint/dart/FramePairDifferentiable.hpp"
+#include "constraint/dart/FrameTestable.hpp"
+#include "constraint/dart/InverseKinematicsSampleable.hpp"
+#include "constraint/dart/JointStateSpaceHelpers.hpp"
+#include "constraint/dart/TSR.hpp"
 #include "constraint/uniform/RnBoxConstraint.hpp"
 #include "constraint/uniform/RnConstantSampler.hpp"
 #include "constraint/uniform/SE2BoxConstraint.hpp"
 #include "constraint/uniform/SO2UniformSampler.hpp"
 #include "constraint/uniform/SO3UniformSampler.hpp"
+
+namespace aikido {
+namespace constraint {
+
+// Add aliases for long class names here.
+
+using uniform::R0BoxConstraint;
+using uniform::R1BoxConstraint;
+using uniform::R2BoxConstraint;
+using uniform::R3BoxConstraint;
+using uniform::R6BoxConstraint;
+using uniform::RnBoxConstraint;
+using uniform::R0ConstantSampler;
+using uniform::R1ConstantSampler;
+using uniform::R2ConstantSampler;
+using uniform::R3ConstantSampler;
+using uniform::R6ConstantSampler;
+using uniform::RnConstantSampler;
+using uniform::SO2UniformSampler;
+using uniform::SO3UniformSampler;
+using uniform::SE2BoxConstraint;
+
+using dart::CollisionFree;
+using dart::CollisionFreeOutcome;
+using dart::FrameDifferentiable;
+using dart::FramePairDifferentiable;
+using dart::FrameTestable;
+using dart::InverseKinematicsSampleable;
+using dart::TSR;
+using dart::createDifferentiableBounds;
+using dart::createProjectableBounds;
+using dart::createTestableBounds;
+using dart::createSampleableBounds;
```

Comment:
It seems `using namespace dart` (in `aikido::constraint` namespace) also works here instead of listing all the classes. But I'm not sure if this is a good practice. @psigen Could you shed some light on here? :smile: